en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md

```diff
@@ -65,6 +64,11 @@ Now we can access our schema on the default GraphQL endpoint, `/graphql`.
 
 Test it out!
 
+> [!NOTE]
+> Note the use of the default arguments on `date`. Fields created from `DBFields`
+> generate their own default sets of arguments. For more information, see
+> [DBFieldArgs](query_plugins#dbfieldargs).
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.